src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -211,6 +211,14 @@ export default class FilterSimpleConditionWidget extends Vue {
     );
   }
 
+  get filteredAvailableVariables(): VariablesBucket | undefined {
+    if (!this.useDateInput || !this.availableVariables) {
```

Comment:
I think it may be easier to have just two set of variables whatever the case:
`availableVariables` and `dateAvailableVariables`, and then use the `inputWidget` to choose which one to use:
```
availableVariablesForInputWidget(inputWidget) {
  switch(inputWidget) {
    case NewInputWidget:
      return this.dateAvailableVariables;
   default:
     return this.availableVariables
}
```

This way, you do not need a computed (useDateInput) that needs to replicate and stay in sync with the widget actually chosen.